db/revert/tables/applications/remove_unique_owner.sql

```diff
@@ -2,6 +2,7 @@
 
 BEGIN;
 
-ALTER TABLE ccbc_public.applications ADD CONSTRAINT applications_owner_key UNIQUE(owner);
+-- commented out becuase it causes errors on `sqitch revert`
+-- ALTER TABLE ccbc_public.applications ADD CONSTRAINT applications_owner_key UNIQUE(owner);
```

Comment:
Do we want to keep this commented out? If we keep this commented out, when someone attempts to revert before this they won't be in the same state as before this was deployed. 

I'm not sure what the proper solution for this should be. Can @wenzowski provide some suggestions? The problem is when the db is populated, this causes the revert to fail.

Reply:
I would suggest to keep it commented out. It does not harm existing or new users as it result in `ccbc_public.applications` without the constraint - which is the requirement to allow multiple applications for the same user.

Reply:
Agreed, I'll go ahead and approve this then